src/components/ErrorBoundry/ErrorBoundry.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import {ERROR_MESSAGE} from '../../const.js';
+
+const ErrorBoundry =(props) =>{
+    const ErrTxt = () =>(
+        <h1>
```

Comment:
It is not a good idea to use h1 here. Also, your ErrorBoundary component doesn't have the necessary logic. It should catch errors if any and display alternative text. But your component doesn't catch any errors. 
There is a link in learn portal to an example of ErrorBoundary component https://reactjs.org/docs/error-boundaries.html

Reply:
got it will fix

Reply:
As I understood I can use only Class component to support ErrorBoundary and state, so I thought  we will work with state within HOOKs/Redux. Official React team not provided Error boundary support for functional component. We can achieve error boundary for functional component using npm package. But I'm not sure. In my case I can only set BOOL variable to true to raise the error.

Reply:
Yes. there is no functional implementation. But it is not a problem to use class component or as you said, some external npm package